src/cirrus/management/commands/manage.py

```diff
@@ -122,7 +131,7 @@ def run_workflow(deployment, timeout, raw, poll_interval):
         timeout=timeout,
         poll_interval=poll_interval,
     )
-    click.echo(json.dump(output, sys.stdout, indent=4 if not raw else None))
+    click.echo(json.dump(output, sys.stdout, indent=4 if not raw else None))  # type: ignore
```

Comment:
I saw there was some other chatter about this line--I think this is a bit of nit, but just throwing out the option to do

```suggestion
    click.echo(json.dumps(output, indent=(4 if not raw else None)))
```

for consistency with other CLI output where we are using `dumps` not `dump`. I'm not sure the possible performance difference here is meaningful one way or the other. I don't think we are really concerned here about memory usage and thus don't want the intermediate string allocation--which probably happens regardless. If that is truly a concern then we'd probably need to be reevaluating the rest of the code in that regard. As a result I'd lobby for this suggested change.